core/payment-driver/gnt/src/gnt/sender.rs

```diff
@@ -759,7 +768,8 @@ impl TransactionSender {
                 me.send(PendingConfirmation {
                     tx_id,
                     tx_hash,
-                    confirmations: 5,
+                    // TODO
```

Comment:
TODO?

Reply:
Forgot to remove the comment, will force push ;)